tools/packaging/kernel/build-kernel.sh

```diff
@@ -188,8 +188,8 @@ get_kernel_frag_path() {
 	# when it reports an error or warning condition. We search for them in the
 	# output to try and fail when we think something has been misconfigured.
 	local not_in_string="not in final"
-	local redefined_string="not in final"
-	local redundant_string="not in final"
+	local redefined_string="redefined"
```

Comment:
Are these changes brought in by kernel merge config script?

Reply:
No, but I think not_in_string/redefined_string/redundant_string  should be set to  different value. I'm not sure why  are they all set to  "not in final" currently.